sbin/get-redisearch

```diff
@@ -64,8 +64,8 @@ else
 		if [[ $nick != bullseye ]]; then
 			nick=ubuntu18.04
 		fi
-	elif [[ $dist == centos || $dist == redhat || $dist == fedora ]]; then
-		if [[ $nick == centos8 ]]; then
+	elif [[ $dist == centos || $dist == redhat || $dist == fedora || $dist == ol ]]; then
```

Comment:
`ol` without `nick == ol8` falls to `rhel7`
is this right? Does this flow exist?

Reply:
Practically, we have two options for CentOS-like systems, rhel7 and rhel8 (note that those are RLEC-style names which we're forced to use in packages). The former will work on most systems (so unless we're a type-8 system - which falls under the category of osnick == centos8/ol8, because we don't distinguish between rhel8 and centos8 on the osnick level).
If you find it a conceptual hell, you're not wrong. It is.